incubator/cassandra/values.yaml

```diff
@@ -2,7 +2,7 @@
 ## ref: https://hub.docker.com/r/library/cassandra/
 image:
   repo: "cassandra"
-  tag: "3.11.2"
+  tag: "3.11.3"
```

Comment:
README should match as well

Reply:
Fixed. Thanks!

Reply:
DCO doesn't allow quick edit from github. I will update the author info later today.

Reply:
updated